lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/BltiEntity.java

```diff
@@ -267,6 +272,50 @@ public LessonEntity getEntity(String ref) {
 	    return null;
     }
 
+	public void preShowItem(SimplePageItem item) {
+		loadContent();
+		if ( content != null ) {
+			Long contentKey = getLong(id);
+			String item_name = item.getName();
+			String item_description = item.getDescription();
+			String content_title = (String) content.get(LTIService.LTI_TITLE);
+			// SAK-43966 - Until the field is added description will always be null
+			String content_description = (String) content.get(LTIService.LTI_DESCRIPTION);
+			// SAK-40044 - The pre-21 workaround is to pass description in settings
+			// once SAK-43996 is done, this can be deleted
+			String content_settings = (String) content.get(LTIService.LTI_SETTINGS);
+			JSONObject content_json = BasicLTIUtil.parseJSONObject(content_settings);
+			String json_description = (String) content_json.get(LTIService.LTI_DESCRIPTION);
+
+			if ( (item_name != null && ! item_name.equals(content_title)) ||
+				(item_description != null && ! item_description.equals(json_description)) || // Remove after SAK-43996
```

Comment:
StringUtils really makes this a lot easier to read.

!StringUtils.equals(item_name, content_title)

Also what is SAK-43996 ? I can't find it in JIRA